client/src/components/QnAWidget/Answer.jsx

```diff
@@ -1,14 +1,34 @@
 import React from 'react';
-const Answer = function(prop) {
 
-  return(
+const Answer = function ({ answer }) {
+  const [helpfull, setHelpfull] = React.useState(false);
+  const [reported, setReported] = React.useState(false);
+  const helphandler = (event) => {
+    event.preventDefault();
```

Comment:
Since it's not a form that do the submission stuff, page won't be refreshed. so event.preventDefault is not needed. So remove event parameter.